ppr-ui/src/composables/mhrRegistration/useHomeOwners.ts

```diff
@@ -48,6 +48,12 @@ export function useHomeOwners (isMhrTransfer: boolean = false) {
     'setMhrTransferHomeOwnerGroups'
   ])
 
+  const executorTrusteeAdmin = [
+    HomeOwnerPartyTypes.EXECUTOR,
+    HomeOwnerPartyTypes.TRUSTEE,
+    HomeOwnerPartyTypes.ADMINISTRATOR
+  ]
```

Comment:
Unless using this elsewhere, i would define within the scope of the function expression it's used.